doc/man3/X509_get_extension_flags.pod

```diff
@@ -78,12 +78,17 @@ The certificate contains an unhandled critical extension.
 
 =item B<EXFLAG_INVALID>
 
-Some certificate extension values are invalid or inconsistent. The
-certificate should be rejected.
+Some certificate extension values are invalid or inconsistent.
+The certificate should be rejected.
 This bit may also be raised after an out-of-memory error while
 processing the X509 object, so it may not be related to the processed
 ASN1 object itself.
 
+=item B<EXFLAG_HASH_INVALID>
```

Comment:
I wonder if a name like `EXFLAG_NO_FINGERPRINT` might make more sense -- the current name feels like it's trying to be very similar to `EXFLAG_INVALID`, but they seem to reflect qualitatively different things -- a property of the certificate vs our ability to process it.

Reply:
Good point - done.